src/test/java/teammates/test/driver/FileHelper.java

```diff
@@ -21,6 +21,7 @@ private FileHelper() {
     /**
      * Reads the file with the specified path as a String.
      */
+    @SuppressWarnings("PMD.UnnecessaryLocalBeforeReturn") // local variable needed for resource cleanup
```

Comment:
Have you considered using a try-finally block? Can do `return sc.useDelimiter("\\Z").next();` in the try and `sc.close();` in the finally.